### ISSUE TITLE
How initialize locomotive when all images are loaded

### ISSUE THREAD

**Issue by Dushyant1295** [DidoMarchet/starter-kit-nuxt-locomotive-scroll#1]
Hi @DidoMarchet  Thanks alot for this you save hours for me before nuxt i used locomotive with barba.js   can you explain how to update locomotive scroll after all images  are loaded

**Comment by DidoMarchet** [comment]
Hi @Dushyant1295!
Glad this repo is useful!

Well, I added this example:
[https://starter-kit-nuxt-locomotive-scroll.netlify.app/image-loads/](https://starter-kit-nuxt-locomotive-scroll.netlify.app/image-loads/) with a delayed image loads. 

No width and height or padding are setted then this is the worst scenario for an image and the refresh works well.

I added in the Locomotive Scroll component a listener
```
this.$nuxt.$on('update-locomotive', () => {
    this?.locomotive?.update()
})
````
and after the image (or images) is load I emit:
```
this.$nuxt.$emit('update-locomotive')
```

I'm doing the same thing explained in Gotcha section.

My advice is to define the wrapper ratio first (some example [https://css-tricks.com/aspect-ratio-boxes/](https://css-tricks.com/aspect-ratio-boxes/)).

Regards, 
Davide
